docs/pages/4.advanced.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Advanced
-description: 'Learn more about the algolia module for your Nuxt application.'
+description: "Learn more about the algolia module for your Nuxt application."
```

Comment:
I think you have linter applying changes to the documentation projects as well that can result in some issues. Could you please revert these changes?

Reply:
Oops yes sorry about that